requirements-test.txt

```diff
@@ -1,5 +1,6 @@
-nose==1.3.4
+docker-py==1.8.1
```

Comment:
I had this same issue/question with `docker-py` and `kazoo`... do these belong here on in `requirements.txt`. Here right?